getting-started/README.md

```diff
@@ -1,71 +1,111 @@
 
-[Requests](../guidelines/requests.md) towards our API should be in `JSON` format and sent using `POST` method. Below is an example request which will return the configuration details for one of our test hotels. You can find out more about this API operation in [Get configuration](../operations/configuration.md#get-configuration).
+- **Mews environments**<br>
+  Mews provides two primary environments:
+  - [Demo environment](../guidelines/environments.md#demo-environment) – for development and testing.
+    - Net pricing and Gross pricing variants simulate different tax regimes used by properties in different jurisdictions.
+  - [Production environment](../guidelines/environments.md#production-environment) – used by live properties after certification.
 
-Endpoint URL:
+- **Mews Operations**<br>
+  You can optionally log into the UI of Mews Operations (used by hotel staff) to understand how API data maps to the product.
 
-`https://api.mews-demo.com/api/connector/v1/configuration/get`
+- **Demo credentials**<br>
+  Mews provides shared tokens and login details for test properties. These are suitable for early development, but subject to [rate limits](../guidelines/requests.md#request-limits).
 
-Request payload:
-```javascript
+> **Want to know more?** For full details, see [Authentication](../guidelines/authentication.md) and [Environments](../guidelines/environments.md).
+
+## Step 1: Set up your environment
+
+To begin testing:
+
+1. **Choose a demo property** from the list in [Demo environments](../guidelines/environments.md#demo-environments).
+   - Use either the Net pricing or Gross pricing variant depending on your target markets.
+2. **Note the tokens** for that property:
+   - `ClientToken`
+   - `AccessToken`
+3. **(Optional)** Log in to [Mews Operations](https://app.mews-demo.com) using the provided email and password to explore the UI and sample data.
+
+You’ll use these credentials to make authenticated API requests.
+
+## Step 2: Make your first API call
+
+Let’s start by calling [Get configuration](../operations/configuration.md#get-configuration), which returns details about the test property — a useful check that your setup is working.
+
+- **Endpoint URL**
+
+```
+https://api.mews-demo.com/api/connector/v1/configuration/get
+```
+
+- **Request body**
+```json
 {
-    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
-    "AccessToken": "7059D2C25BF64EA681ACAB3A00B859CC-D91BFF2B1E3047A3E0DEC1D57BE1382",
-    "Client": "NameOfYourCompanyOrApplication"
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "7059D2C25BF64EA681ACAB3A00B859CC-D91BFF2B1E3047A3E0DEC1D57BE1382",
+  "Client": "NameOfYourCompanyOrApplication"
 }
 ```
 
-If the [response](../guidelines/responses.md) looks something like this, you've successfully made your first request!
-```javascript
+- **Expected response**  
+If successful, the API will return property details:
```

Comment:
idea: what about to say it's return 200 response code with property details (without putting the response here). It's more simple imho.